sdk/data/aztables/go.mod

```diff
@@ -1,10 +1,29 @@
 module github.com/Azure/azure-sdk-for-go/sdk/data/aztables
 
-go 1.16
+go 1.18
+
+replace github.com/Azure/azure-sdk-for-go/sdk/azcore => ../../azcore
```

Comment:
Are we going to push a tagged version of azcore?

Reply:
Yes, once core is tagged I'll update this.